Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -492,116 +498,125 @@ class BocList
     }
     // Add diganostic metadata for web testing framework (actually: should only be rendered with IRenderingFeatures.EnableDiagnosticMetadata on)
-    tableContainer.attr('data-boclist-has-fake-table-head', 'true');
-  
-    var table = scrollableContainer.children('table').first();
-  
-    var fakeTable = $('<table/>');
-    fakeTable.attr({
-        'role' : 'none',
-        'class' : table.attr('class'), 
-        cellPadding: 0,
-        cellSpacing: 0
-      });
-    fakeTable.css({ width: '100%' });
-  
-    var realTableHead = table.children('thead').first();
-    var fakeTableHead = realTableHead.clone(true, true);
-    realTableHead.attr({
-      'aria-hidden': 'true',
-      'role': 'none'
+    tableContainer.setAttribute('data-boclist-has-fake-table-head', 'true');
+  
+    var table = scrollableContainer.querySelector(':scope>table')!;
+  
+    var fakeTable = document.createElement('table');
+    fakeTable.setAttribute('role', 'none');
+    table.classList.forEach(className => { fakeTable.classList.add(className) });
+    fakeTable.setAttribute('cellPadding', '0');
+    fakeTable.setAttribute('cellSpacing', '0');
+    fakeTable.style.width = '100%';
+
+    var realTableHead = table.querySelector(':scope>thead')!;
+    var fakeTableHead = realTableHead.cloneNode(true) as HTMLElement;
+    realTableHead.setAttribute('aria-hidden', 'true')
+    realTableHead.setAttribute('role', 'none')
+    realTableHead.querySelectorAll('*[role]').forEach(element =>
+    {
+      element.setAttribute('role', 'none');
     });
-    realTableHead.find ('*[role]').attr ({ 'role' : 'none' });
     fakeTable.append(fakeTableHead);
   
-    var fakeTableHeadWidthContainer = $('<div/>').attr ({ 'role' : 'none' });
+    var fakeTableHeadWidthContainer = document.createElement('div');
+    fakeTableHeadWidthContainer.setAttribute('role', 'none');
     fakeTableHeadWidthContainer.append(fakeTable);
   
-    var fakeTableHeadContainer = $('<div/>').attr({ 'role': 'none', 'class': 'bocListFakeTableHead' });
-    fakeTableHeadContainer.hide();
+    var fakeTableHeadContainer = document.createElement('div');
+    fakeTableHeadContainer.setAttribute('role', 'none');
+    fakeTableHeadContainer.classList.add('bocListFakeTableHead');
+    fakeTableHeadContainer.style.display = 'none';
     fakeTableHeadContainer.append (fakeTableHeadWidthContainer);
   
-    realTableHead.find('*').each(function (this: HTMLElement) { $(this).removeAttr('id').attr({ tabIndex: -1 }).attr({ tabIndex: -1 }); });
+
+    realTableHead.querySelectorAll('*').forEach(element =>
+    {
+      element.removeAttribute('id');
+      element.setAttribute('tabIndex', '-1');
+    });
   
     scrollableContainer.before(fakeTableHeadContainer);
   
     // sync checkboxes
-    var checkboxes = fakeTableHead.find("th input:checkbox");
-    checkboxes.click(function (this: HTMLElement)
+    var checkboxes = fakeTableHead.querySelectorAll("th input[type=checkbox]");
```

Comment:
We should probably use `<HTMLInputElement>` here so we can remove following cast